Dockerless Guide.md

```diff
@@ -0,0 +1,135 @@
+
+```toml
+[Unit]
+Description=XUD
+
+[Service]
+User=xud
+Group=xud
+Type=simple
+Environment=NODE_ENV=production
+ExecStart=/home/xud/xud/bin/xud --mainnet
+KillMode=process
+KillSignal=SIGINT
+```
+
+# LND (BTC)
+
+Follow the [lnd installation guide](https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md#installing-lnd).
+
+## Daemonize `lnd`
+
+If you want to daemonize `lnd`, so that it starts on boot without needing its own terminal, you can do this using `systemd`:
+
+```toml
+[Unit]
+Description=LND
+
+[Service]
+User=xud
+Group=xud
+Type=simple
+ExecStart=/home/xud/lnd/bin/lnd --bitcoin.mainnet
+KillMode=process
+KillSignal=SIGINT
+```
+
+# Connext
+
+Follow the [docs](https://github.com/connext/vector#quick-start), you want the **`node`: vector node + database** stack.
+
+## Daemonize `connext`
+
+If you want to daemonize `connext`, so that it starts on boot without needing its own terminal, you can do this using `systemd`:
+
+```toml
+[Unit]
+Description=Connext
+
+[Service]
+User=xud
+Group=xud
+Type=simple
+Environment="NODE_ENV=production"
+Environment="CONNEXT_NODE_URL=https://connext.boltz.exchange"
+Environment="CONNEXT_ETH_PROVIDER_URL=http://eth.kilrau.com:41007"
+Environment="LEGACY_MODE=true"
+WorkingDirectory=/home/xud/connext/
+ExecStart=node /home/xud/connext/build/src/index.js
+KillMode=process
+KillSignal=SIGINT
+```
+
+# Tor
+
+You can install tor via `sudo apt install tor` on most linux distros nowadays, just make sure [the version is fairly recent](https://github.com/torproject/tor/releases). If not, consult the [tor installation guides](https://2019.www.torproject.org/docs/installguide.html.en). Run `tor --version` to verify the tor process is running.
+
+
+# Putting it all together
+
+Create the following `xud.conf` in `/home/xud/.xud`:
+```toml
+mainnet = true
+
+[p2p]
+tor = true
+torport = 9050
+
+[connext]
+disable = false
+host = "localhost"
+port = 5000
+webhookhost = "localhost"
+webhookport = 8887
+
+[lnd.BTC]
+disable = false
+host = "localhost"
+certpath = "/home/xud/.lnd/tls.cert"
+macaroonpath = "/home/xud/.lnd/admin.macaroon"
+
+[lnd.LTC]
+disable = true
+```
+
+For convenience, consider adding `alias xucli='/home/xud/xud/bin/xucli -p 8886'` to the xud user's `.bashrc` and source it. Then restart `xud` once (e.g. with `systemctl restart xud`) and try running `xucli getinfo`, which should return with an overview of xud's, as well as lnd's and connext status.
+
+Ping us in the help channel of our [Discord server](https://discord.gg/YgDhMSn) for support.
+
+# Tips 'n Tricks
+
+* When installing on a Raspberry Pi you might see `Unexpected error during initialization`. [Here](https://github.com/ExchangeUnion/xud/issues/1199#issuecomment-527819108) the solution.
+* If you see an `install error` when installing via `npm install`, try `npm install --production` & `npm install -g typescript`.
```

Comment:
```suggestion
* If you see an `install error` when installing via `npm install`, try `npm install --production` & `npm install typescript`.
```

TypeScript does not have to be installed globally. Just inside the package is enough

Reply:
Fixed